src/test/java/me/itzg/tsdbcassandra/services/DownsampleProcessorTest.java

```diff
@@ -51,8 +51,8 @@
   @TestConfiguration
   static class TestConfig {
     @Bean
-    public Scheduler downsampleScheduler() {
-      return Schedulers.immediate();
+    public TaskScheduler downsampleTaskScheduler() {
```

Comment:
this doesn't actually excercise the real scheduler, right?  Should it have any tests?

Reply:
I'll drop a TODO assertion-fail for now, because yes, it needs a test, but it's an odd one to test.